### PR TITLE
Preferences updates for v7.9.1

### PR DESCRIPTION
**Pull request by pryrt**
Add descriptions for new preference-settings
1. **Preferences > General > Tab Bar: ☐ Alternate icons**
2. **Preferences > MISC. > ☐ Set Save dialog file extension filter...**

Reference the second from the "new style dialog" setting description.

### CODE REVIEW

**Comment by sasumner** [comment]
Apologies @pryrt ; these are my features, I should document them.
I have a note-to-self to do some manual work; I should read my own reminder notes.

**Comment by pryrt** [comment]
@sasumner , 
No problem. The real question is whether my description was good enough to be approved... *hint, hint* ;-)

File: content/docs/preferences.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ These affect the user interface (localization, toolbar, tab bar, and more).
     * `☐ Multi-line`: if there are enough tabs, they will wrap to a second line
     * `☐ Vertical`: have the tabs on their side on the left, rather than along the top
     * `☐ Reduce`: make the tab bar vertical region and font size smaller
+    * `☐ Alternate icons`: change the saved/edited/read-only icons from blue/red/grey disk-icons to checkmark/pencil/lock symbols
```
Comment (sasumner) [inline content/docs/preferences.md:33]: Append?:
```
, respectively; the alternate icon feature is mainly to improve the user experience for visually-impaired users.
```

File: content/docs/preferences.md [hunk 2]
```diff
@@ -313,6 +314,7 @@ A variety of settings that didn't fit elsewhere
     * `☐ Update silently`: instead of prompting, will automatically reload the file from disk
     * `☐ Scroll to the last line after update`: will scroll to the end of the file after reloading from disk (otherwise, the cursor and scrolled-location stays where it was before the update)
 * `☐ Enable Notepad++ auto-updater`: will automatically download updates from the official website, once the development team has decided it's time to push an update to users.  If disabled, you will have to manually download the installer from the official website yourself.
+* `☐ Set Save dialog file extension filter... for Normal Text`: when this is enabled and the current file is set to Normal Text, the **Save As** dialog's filter will default to `*.*` and require you to type an extension; when disabled, the filter will default to `*.txt` and will auto-append the `.txt` extension if you don't type an extension or choose a different filter.  (Regardless of the state of this option, you can always select a different filter manually in the **Save As** dialog.)
```
Comment (sasumner) [inline content/docs/preferences.md:317]: "and the current file is set to Normal Text" ---> "and the current file's language is set to Normal Text" ??

Somehow make a note that this is only for "untitled files", you know, "new 2" and the like.  (If you do "Save As" on an existing "test.txt", it won't set the filter to \*.\*)

**Comment by sasumner** [comment]
> The real question is whether my description was good enough to be approved... _hint, hint_ ;-)

I know I can always comment on things, but when I see:

`pryrt requested a review from notepad-plus-plus/notepad-user-manual-admin 3 hours ago`

I don't feel like I'm specifically wanted.  :-)

File: content/docs/preferences.md (hunk 1 above)
Comment (pryrt) [reply content/docs/preferences.md]: added

File: content/docs/preferences.md (hunk 2 above)
Comment (pryrt) [reply content/docs/preferences.md]: clarified

**Review by sasumner — APPROVED** [review]

**Pull request merged**